src/app/repo-detail/repo-detail.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { RepoService } from '../repo.service';
+import { Repository } from '../repository';
+
+@Component({
+  selector: 'app-repo-detail',
+  templateUrl: './repo-detail.component.html',
+  styleUrls: ['./repo-detail.component.css']
+})
+export class RepoDetailComponent implements OnInit {
+  @Input() repo?: Repository;
+  name?: string;
+  repoDetails: string[][] = [];
+  
+  constructor(
+    private route: ActivatedRoute,
+    private repoService: RepoService
+  ) { }
+
+  ngOnInit(): void {
+    this.getRepo();
+  }
+
+  getRepo(): void {
```

Comment:
Tiny nit, but I would expect any function that is prefixed with `get` to return whatever is in the name (in this case, a repo). Could you think of another name that would eliminate any confusion?

Reply:
done!